packages/actions/src/helpers/utils.ts

```diff
@@ -0,0 +1,33 @@
+/**
```

Comment:
These two methods below are used only in the `setup` command. Therefore, let's move everything `src/core/setup` as I did for the `auth` command methods (`getNewOAuthTokenUsingGithubDeviceFlow()`, `getCurrentFirebaseAuthUser()`, `signInToFirebaseWithGithubToken()`).

This will avoid having a shared `helpers/` module which is only used in one command!